src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
@mikem8361 this change doesn't remove the signal handlers settings for the debugger shared libs since they are not loaded via the LoadLibrary:
https://github.com/dotnet/coreclr/blob/master/src/ToolBox/SOS/lldbplugin/soscommand.cpp#L105
Or am I missing something?